src/renderer/src/components/song/song-item/SongItem.tsx

```diff
@@ -54,17 +54,27 @@ const SongItem: Component<SongItemProps> = ({
 
   return (
     <div
-      class="song-item"
+      class={`relative isolate select-none rounded-md ${
+        selectedSong().path === song.path
+          ? "data-[active=true]:outline data-[active=true]:outline-2 data-[active=true]:outline-accent data-[active=true]:after:bg-black/30"
+          : ""
+      } hover:after:bg-overlay`}
       data-active={selectedSong().path === song.path}
       ref={item}
       data-url={song.bg}
       onContextMenu={showMenu}
     >
-      <SongImage class="song-item__image" src={song.bg} group={group} />
+      <SongImage
+        class="absolute inset-0 z-[-1] h-full w-full rounded-md bg-cover bg-center bg-no-repeat opacity-50 data-[active=true]:opacity-100"
```

Comment:
The `data-[active=true]` doesn't work here because the attr is on the parent component, you need to use a group to do that, however, a better solution is:

```tsx
class="absolute inset-0 z-[-1] h-full w-full rounded-md bg-cover bg-center bg-no-repeat opacity-50"
        classList={{
          "opacity-100": selectedSong().path === song.path,
        }}
```